app/(app)/home/about.tsx

```diff
@@ -1,34 +1,108 @@
+import { useTheme } from '@react-navigation/native';
+import { Image } from 'expo-image';
 import { Stack } from 'expo-router';
 import React, { useState } from 'react';
-import { View } from 'react-native';
+import type { ListRenderItemInfo } from 'react-native';
+import { Dimensions, FlatList, StyleSheet, View } from 'react-native';
+import Space from '../../../components/common/Space';
 import StyledText from '../../../components/common/StyledText';
 import MainContainer from '../../../components/container/MainContainer';
 import HeaderRight from '../../../components/headers/HeaderRight';
 import GoogleSignInModal from '../../../components/modals/GoogleSignInModal';
+import { blurhash } from '../../../config/constants';
+import type { OrganizingTeamMember } from '../../../global';
+import { OrganizingTeam } from '../../../mock/organizingTeam';
 
-// TODO: About page - implement about event page
-/**
- * -  should display an image, description and a grid view of organizing team, and an organizers card
- * -  consider reusing the image component that was used in the speakers list
- * -  clicking on the team member image should navigate to the [speaker] page
- */
-
-// TODO: Use data from mock/organizers.ts
+const { width } = Dimensions.get('window');
 
 const About = () => {
+  const { colors } = useTheme();
   const [signInModalVisible, setSignInModalVisible] = useState<boolean>(false);
 
   const showSignInModal = () => {
     setSignInModalVisible(true);
   };
+  const OrganizingIndividuals = OrganizingTeam.data.filter((item) => item.type === 'individual');
   return (
     <MainContainer preset="scroll">
       <Stack.Screen
         options={{
           headerRight: () => <HeaderRight handlePress={showSignInModal} />,
         }}
       />
-      <StyledText>about</StyledText>
+      <View style={styles.container}>
+        <Image
+          style={styles.image}
+          source="https://res.cloudinary.com/khariokitony/image/upload/v1690514752/Droidcon-organizers.png"
+          placeholder={blurhash}
+          contentFit="contain"
+        />
```

Comment:
This image doesn't fill screen width. I don't know why. I thought maybe it was padding or something like that. What could be the reason?

Reply:
this has been tested on both android and ios. and emulators too. it displays full width. could you attach a screenshot showing the bug

Reply:
> 
![Screenshot_1690629621](https://github.com/droidconKE/droidconKE2023ReactNative/assets/28436092/93a8822a-303c-4e2d-aac7-8d342be83c08)
see the horizontal padding, between the image

Reply:
When I open about screen on the emulator ( Pixel 2 API 30 ) it sometimes crashes. But when I open About screen on Samsung Galaxy A30, the app crashes

Reply:
@kharioki  this issue still occurs